packages/plugin-core/src/commands/RandomNote.ts

```diff
@@ -0,0 +1,81 @@
+import { NoteProps, NoteUtils } from "@dendronhq/common-all";
+import { Uri, window } from "vscode";
+import { DENDRON_COMMANDS } from "../constants";
+import { VSCodeUtils } from "../utils";
+import { getWS } from "../workspace";
+import { BasicCommand } from "./base";
+import _ from "lodash";
+
+type CommandOpts = {};
+
+type CommandInput = {};
+
+type CommandOutput = void;
```

Comment:
a convention we've been following is to return the note as an output. makes it easy to test

Reply:
cool, will adjust